images/step-cli/tests/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
```

Comment:
Can we test functional usage for this image?

Reply:
I tried but it prompts for input for a password from the TTY. I couldn't figure a way around it.
Here's what I was originally trying and gave up doing:

```
$ docker run -it k3d-k3d.localhost:5005/step-cli@sha256:0fa2855518b304119a998177167984921b4c2fa564a41aff2a96c2e7a76e321d ca init --deployment-type=standalone --name=step-ca --dns=127.0.0.1 --address=127.0.0.1:8443 --provisioner=step-ca```

Reply:
Found a decent working functional test.